modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,32 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="filelog-receiver_{context}"]
+=== Filelog Receiver
+
+:FeatureName: The Filelog Receiver
+include::snippets/technology-preview.adoc[]
+
+The Filelog Receiver tails and parses logs from files.
```

Comment:
```suggestion
The Filelog Receiver tails and parses logs from files.

:FeatureName: The Filelog Receiver
include::snippets/technology-preview.adoc[]
```
I would recommend putting the abstract/introductory sentence before the tech preview snippet. it's useful to get some information about what the component is before reading about its support status.

Reply:
Thank you. I'll swap these lines as part of a global revision PR for this file. There are more lines like these throughout the file, and those are outside of this PR, so I'll fix it globally for the whole file in a single PR.